common/src/api/external/mod.rs

```diff
@@ -490,21 +488,59 @@ impl JsonSchema for RoleName {
     }
 }
 
-/// A count of bytes, typically used either for memory or storage capacity
+/// This is used to express memory or storage capacity.
 ///
 /// The maximum supported byte count is [`i64::MAX`].  This makes it somewhat
-/// inconvenient to define constructors: a u32 constructor can be infallible, but
-/// an i64 constructor can fail (if the value is negative) and a u64 constructor
-/// can fail (if the value is larger than i64::MAX).  We provide all of these for
-/// consumers' convenience.
-// TODO-cleanup This could benefit from a more complete implementation.
-//
-// The maximum byte count of i64::MAX comes from the fact that this is stored in
-// the database as an i64.  Constraining it here ensures that we can't fail to
-// serialize the value.
-#[derive(Copy, Clone, Debug, Deserialize, Serialize, JsonSchema, PartialEq)]
+/// inconvenient to define constructors: a u32 constructor can be infallible,
+/// but an i64 constructor can fail (if the value is negative) and a u64
+/// constructor can fail (if the value is larger than i64::MAX).  We provide
+/// all of these for consumers' convenience.
+///
+/// The maximum byte count of i64::MAX comes from the fact that this is stored
+/// in the database as an i64.  Constraining it here ensures that we can't fail
+/// to serialize the value.
+///
+/// TODO-cleanup This could benefit from a more complete implementation.
```

Comment:
This documentation seems inaccurate next to the `JsonSchema` implementation below?

Reply:
I'm going to pull this out and do it in another PR--too many snags.